app/assets/javascripts/oxalis/view/right-menu/tree_hierarchy_view.js

```diff
@@ -130,12 +135,28 @@ class TreeHierarchyView extends React.PureComponent<Props, State> {
 
   onSelectTree = evt => {
     const treeId = evt.target.dataset.id;
-    this.props.onSetActiveTree(parseInt(treeId, 10));
+    if (evt.ctrlKey) {
+      this.props.handleTreeSelect(parseInt(treeId, 10));
```

Comment:
I'd just change line 137 to `const treeId = parseInt(evt.target.dataset.id, 10)` to avoid duplication. Then you can just pass `treeId` here and in the else branch.